src/layout/flex.rs

```diff
@@ -1,19 +1,183 @@
 use strum::{Display, EnumString};
 
-/// Option for layout flex justify content
+/// Defines the options for layout flex justify content in a container.
 ///
-/// This controls how the space is distributed when the constraints are satisfied.
+/// This enumeration controls the distribution of space when layout constraints are met.
+///
+/// - `StretchLast`: Fills the available space within the container, putting excess space into the
+///   last element.
+/// - `Stretch`: Always fills the available space within the container.
+/// - `Start`: Aligns items to the start of the container.
+/// - `End`: Aligns items to the end of the container.
+/// - `Center`: Centers items within the container.
+/// - `SpaceBetween`: Adds excess space between each element.
+/// - `SpaceAround`: Adds excess space around each element.
 #[derive(Copy, Debug, Default, Display, EnumString, Clone, Eq, PartialEq, Hash)]
 pub enum Flex {
+    /// Fills the available space within the container, putting excess space into the last element.
```

Comment:
Add a note about this being the default for matching existing behavior of ratatui / tui apps?

Reply:
I added a note documenting the default.